### ISSUE TITLE
VideoTile is closing while turning off the video button

### ISSUE THREAD

**Issue by dharshinithilagar** [aws/amazon-chime-sdk-js#2539]
### What happened and what did you expect to happen?

Hi,

I was working  in [amazon-chime-sdk-js](https://github.com/aws/amazon-chime-sdk-js). I need to turn on and off the video as per the requirement. 

I have added the given code for on and off video from this github repository. I can on the video and while offing it closes the video tile that has both customer and agent video from the webpage. So kindly assist me for not closing the video tile .

### Have you reviewed our existing documentation?

- [x] GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [x] FAQs [page](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html)
- [x] README [page](https://github.com/aws/amazon-chime-sdk-js#amazon-chime-sdk-for-javascript)
- [x] Feature-specific [guides](https://github.com/aws/amazon-chime-sdk-js#guides)

### Reproduction steps

async function initializaMeetingSession() {
  const logger = new ConsoleLogger('SDK', LogLevel.OFF);
  session = new DefaultMeetingSession(
    new MeetingSessionConfiguration(
      meeting,
      agentAttendee,
    ),
    logger,
    new DefaultDeviceController(logger),
  );

  session.audioVideo.addObserver(observer);

  const firstAudioDeviceId = (await session.audioVideo.listAudioInputDevices())[0].deviceId;
  await session.audioVideo.startAudioInput(firstAudioDeviceId);
  
  const firstVideoDeviceId = (await session.audioVideo.listVideoInputDevices())[0].deviceId;
  await session.audioVideo.startVideoInput(firstVideoDeviceId);

  session.audioVideo.bindAudioElement(domElement('help-desk-audio'));

  session.audioVideo.start();
}
async function stopvideo() {
  console.log('Video is stoping',session);
  if (session) {
    //session.audioVideo.stop();
    //session.audioVideo.stopLocalVideoTile();
    document.getElementById('video_on').style.display = 'block';
    document.getElementById('video_off').style.display = 'none';
    await session.audioVideo.stopVideoInput();
    //session.audioVideo.removeLocalVideoTile();
  }

}

async function startvideo() {
  console.log('Video is starting',session);
  if (session) {
    
    const firstVideoDeviceId = (await session.audioVideo.listVideoInputDevices())[0].deviceId;
    document.getElementById('video_on').style.display = 'none';
    document.getElementById('video_off').style.display = 'block';
    await session.audioVideo.startVideoInput(firstVideoDeviceId);
  }

### Amazon Chime SDK for JavaScript version

^3.6.0

### What browsers are you seeing the problem on?

chrome

### Browser version

Version 108.0.5359.125 (Official Build) (64-bit)

### Browser console logs

[localhost-1672817449821.log](https://github.com/aws/amazon-chime-sdk-js/files/10342202/localhost-1672817449821.log)

**Comment by ltrung** [comment]
@dharshinithilagar Hi, Sorry could you rephrase the issue you are having? Do you mean in a 1:1 video between a customer and an agent if when you turn off the local video, it also turns off the other video?

**Comment by dharshinithilagar** [comment]
Hi @ltrung , 
I was connecting agent and customer via video call. In this from agent side, I customized to on and off video using buttons in the tile. But from agent side when I try to switch off the video, it not only turning off the video but it closes whole video tile.
I have attached the implement code in above

**Comment by xuesichao** [comment]
Hi @dharshinithilagar , `stopVideoInput()` API will auto trigger `stopLocalVideoTile()` API to unbind the video element during meeting.

**Comment by devalevenkatesh** [comment]
The concept of video in Amazon Chime SDK for JavaScript (JS SDK) revolves around three things:
1. The video stream that is acquired from your selected video input device. You use `startVideoInput`, `stopVideoInput` to start stop the stream along with the video tile which is explained more in point 3.
2. The HTML video element you bind your video stream to, for this you have to use `bindVideoElement` API.
3. The `VideoTile` and `VideoTileState`. This is specific to JS SDK and simply a JS class and should not be confused with HTML video element. In JS SDK, 1 video tile can be bound to only 1 video element. Probably in your case you are binding same HTML video element to two video tiles and thus stopping one also stops the other. This is not clear from the code you have here. To achieve binding different video elements to different video tiles you need to use two different HTML video elements with different `id` attributes. Once bounded then you can start/stop the video input.

Please refer to https://github.com/aws/amazon-chime-sdk-js#video in the README for more information on the APIs along with above information in previous comments.

You can also refer to the JS SDK meeting demo for reference implementation on these: https://github.com/aws/amazon-chime-sdk-js/tree/main/demos/browser

Since its been a week, I am resolving this issue due to no activity. If you are still running into issues feel free to re-open it.
